pyproject.toml

```diff
@@ -0,0 +1,208 @@
+    {name = "pymc-devs", email = "pymc.devs@gmail.com"}
+]
+description = "Optimizing compiler for evaluating mathematical expressions on CPUs and GPUs."
+readme = "DESCRIPTION.txt"
+license = {file = "LICENCE.txt"}
+classifiers = [
+    "Development Status :: 6 - Mature",
+    "Intended Audience :: Education",
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python",
+    "Topic :: Software Development :: Code Generators",
+    "Topic :: Software Development :: Compilers",
+    "Topic :: Scientific/Engineering :: Mathematics",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX",
+    "Operating System :: Unix",
+    "Operating System :: MacOS",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+]
+
+keywords = [
+    "pytensor",
+    "math",
+    "numerical",
+    "symbolic",
+    "blas",
+    "numpy",
+    "autodiff",
+    "differentiation",
+]
+dependencies = [
+    "setuptools>=48.0.0",
+    "scipy>=0.14",
+    "numpy>=1.17.0",
+    "filelock",
+    "etuples",
+    "logical-unification",
+    "miniKanren",
+    "cons",
+    "typing_extensions",
+]
+
+[project.urls]
+homepage = "https://github.com/pymc-devs/pytensor"
+repository = "https://github.com/pymc-devs/pytensor"
+documentation = "https://pytensor.readthedocs.io/en/latest/"
+
+
+[project.scripts]
+pytensor-cache = "bin.pytensor_cache:main"
+
+[project.optional-dependencies]
+complete = [
+    "pytensor[jax]",
+    "pytensor[numba]",
+]
+development = [
+    "pytensor[complete]",
+    "pytensor[tests]",
+    "pytensor[rtd]"
+]
+tests = [
+    "pytest",
+    "pre-commit",
+    "pytest-cov>=2.6.1",
+    "coverage>=5.1",
+]
+rtd = [
+    "sphinx>=1.3.0",
+    "sphinx_rtd_theme",
+    "pygments",
+    "pydot",
+    "pydot2",
+    "pydot-ng",
+]
+jax = [
+    "jax",
+    "jaxlib",
+]
+numba = [
+    "numba>=0.55",
+    "numba-scipy>=0.3.0"
+]
+
+[tool.setuptools.packages.find]
+exclude = [
+    "tests",
+    "tests.*"
+]
+
+[tool.setuptools.package-data]
+"*" = [
+    "*.txt",
+    "*.rst",
+    "*.cu",
+    "*.cuh",
+    "*.c",
+    "*.sh",
+    "*.pkl",
+    "*.h",
+    "*.cpp",
+    "ChangeLog",
+    "c_code/*",
+]
+pytensor = [
+    "py.typed"
+]
+"pytensor.misc" = [
+    "*.sh"
+]
+"pytensor.d3viz" = [
+    "html/*",
+    "css/*",
+    "js/*",
+]
+
+[tool.coverage.run]
+omit = [
+    "pytensor/_version.py",
+    "tests/*",
+    "pytensor/assert_op.py",
+    "pytensor/graph/opt.py",
+    "pytensor/graph/opt_utils.py",
+    "pytensor/graph/optdb.py",
+    "pytensor/graph/kanren.py",
+    "pytensor/graph/unify.py",
+    "pytensor/link/jax/jax_linker.py",
+    "pytensor/link/jax/jax_dispatch.py",
+    "pytensor/graph/toolbox.py",
+    "pytensor/scalar/basic_scipy.py",
```

Comment:
Some of these get deleted in your other PR, right?

We should remember to update this after rebasing the other PR

Reply:
yeah, one of these PRs goes first